nilearn/image/image.py

```diff
@@ -665,3 +665,68 @@ def threshold_img(img, threshold, mask_img=None):
     threshold_img = new_img_like(img, img_data, affine)
 
     return threshold_img
+
+
+def math_img(formula, **imgs):
+    """Interpret a string formula using niimg in named parameters.
+
+    Parameters
+    ----------
+    formula: string
+        The mathematical formula to apply to image internal data.
+    imgs:
+        Keyword arguments corresponding to the variables in the formula.
+
+    Example
+    -------
```

Comment:
> Are you ok with apply_math_operators.py ?

example should start with ```plot_....```

> Should I just copy the example I wrote in the doctring to this file ?

Something simple and different example which shows how to use this function on real Nifti images. So that user gets easy going taking this function.